library/ui-styles/src/main/res/drawable/bg_carousel_page_1.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
     <gradient
+        android:angle="@integer/rtl_mirror_flip"
```

Comment:
@bmarty the rtl gradient is fixed by making use of the `rtl_mirror_flip`, ltr = 0 rtl = 180